content/technical_challenges.tex

```diff
@@ -258,45 +264,54 @@ \subsection{Dynamic Ball handling Challenge}
-        A pass counts as a valid pass if the ball stops in front of the receiving robot towards the opponent's goal. The ball has to stop in the vicinity of the receiving robot (at max \qty{1}{\metre} distance between receiver and ball, see~\cref{fig:ball_handling_positions}).
+        In \texttt{Playing} the following happens: The 1st attacker plays the ball towards the 2nd or 3rd attacker while he is under attack by the 1st defender.
+        A pass counts as a valid pass if the ball stops in front of the receiving robot towards the next target either 2nd receiver or opponent's goal. The receiving robot should not have to move backwards, to play the ball. When the ball stops, it has to be in the vicinity of the receiving robot (at max approx. \qty{1}{\metre} distance between receiver and ball, see~\cref{fig:ball_handling_positions}).
         The 1st defender does not walk back in its own half and the goalkeeper remains on the goal line and is not allowed to dive. The objective of the defending team is to intercept the passes, see stopping criteria.
 
         After the 2nd or 3rd attacker received the ball, and the ball is in the defender's half, it gets attacked by the 2nd defender. The task of the 2nd or 3rd attacker is now to pass towards the 3rd or 2nd attacking robot, which than shots onto the goal.
 
-        If a defender pushes an attacking robot, the attacking team gets a time bonus of \qty{15}{\second} and the pushing robot will be removed from this run.
+        All rules from the normal gameplay still apply. Only the standard removal penalty time gets extended to \qty{240}{\second}. 
+        If a defender pushes an attacking robot, the attacking team gets a time bonus of \qty{15}{\second}.
 
     \subsubsection{Challenge Scoring}
 
-        For each run the referee measures the execution time from initial whistle until the run is stopped by one of the following criteria:
+        In each phase of a run the GameController measures the execution time from initial whistle until the run is stopped by one of the following criteria:
 
         \begin{itemize}
             \item A defender touches the ball.
-            \item Ball leaves the field.
+            \item An attacking robot leaves the field.
+            \item Ball leaves the field outside the defending goal area.
             \item An attacker pushes.
             \item A run exceeds \qty{4}{\minute} execution time.
-            \item A goal is scored.
+            \item A goal is scored after three robots have played the ball.
+            \item Attacking team is not communicating with the GameController.
         \end{itemize}
 
-        A score for the run will be calculated based on the following rules:
+        In the case that a goal gets scored before three robots played the ball, or the ball leaves the field within the defending goal area, the ball gets placed on the closest goal kick spot.
```

Comment:
What is meant with "played" only touched or are the passes meant?

Reply:
A phrased it now differently